css/index.css

```diff
@@ -2,16 +2,28 @@
 This imports the `reset.css` file */
 @import url("./reset.css");
 
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  }
```

Comment:
This doesn't look well formatted. Is there any reason Linter missed this??. In any case please run all formatting scripts

Reply:
### I really appreciate this comment, i will attend to it as soon as possible.